clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -1001,6 +1007,14 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
     return create<mlir::cir::GetRuntimeMemberOp>(loc, resultTy, objectPtr,
                                                  memberPtr);
   }
+
```

Comment:
Add comment to both functions explaining what they do, they are a bit less obvious than most on this file.